src/test/java/org/zalando/fahrschein/NakadiReaderDeserializationTest.java

```diff
@@ -109,8 +109,9 @@ public String getName() {
 
         @JsonCreator
         public CustomerChanged(@JsonProperty("metadata") Metadata metadata, @JsonProperty("data_type") String dataType, @JsonProperty("data_op") DataOperation dataOp, @JsonProperty("data") Customer data) {
-            super(metadata, dataType, dataOp, data);
+            super(metadata, DATA_TYPE, dataOp, data);
```

Comment:
`DATA_TYPE` should always equal `dataType` here, but it looks a bit strange to ignore the parameter.